src/Sulu/Bundle/ResourceBundle/Resource/FilterManager.php

```diff
@@ -467,8 +507,28 @@ public function getFeaturesForContext($context)
     }
 
     /**
+     * Removes conditions from condition groups when they are not in the given array
+     *
+     * @param ConditionGroupEntity $conditionGroup
+     * @param array $conditionIds
+     */
+    protected function removeNonExistentConditions(
+        $conditionGroup,
+        $conditionIds
```

Comment:
Because I don't have the conditions to remove at this point? Collecting them happens in this function and then they will be removed immediately. Otherwise I would have to collect them and remove them separately and have to loops?